src/borg/testsuite/archiver/create_cmd.py

```diff
@@ -731,6 +772,42 @@ def test_common_options(self):
         log = self.cmd(f"--repo={self.repository_location}", "--debug", "create", "test", "input")
         assert "security: read previous location" in log
 
+    def test_hashing_time(self):
+        def extract_hashing_time(borg_create_output):
+            borg_create_output = borg_create_output.strip().splitlines()
+            borg_create_output = [line.split(":", 1) for line in borg_create_output]
+            hashing_time = [line for line in borg_create_output if line[0] == "Time spent in hashing"].pop()
+            hashing_time = hashing_time[1]
+            hashing_time = float(hashing_time.removesuffix(" seconds"))
+            return hashing_time
+
+        # Test case set up: create a repository and a file
+        self.cmd(f"--repo={self.repository_location}", "rcreate", "--encryption=none")
+        self.create_regular_file("testfile", size=6000000, contents=randbytes(6000000))
```

Comment:
you don't need to give size if you directly give the contents.

Reply:
btw, guess the root cause for that you need a rather large file to detect `> 0.0` is in `format_timedelta` which needs at least 10ms for that:

https://github.com/borgbackup/borg/blob/3715e327a6ea1306b7fbd3c37f5bdec38b0439bb/src/borg/helpers/time.py#L102

Reply:
What should I do about it?

Reply:
Btw, you are right. I modified the value of txt to `"%.3f seconds" % s` and the test passes with 1MB of data.
But I don't think I should push that modification right?

Reply:
Yeah, guess we should do that modification.

It could well be that we run into problems again after doing the fix to only measure real read/chunk time (without all the processing) and would need to increase the amount of data if we don't do that modification.

Considering that the next unit below s is ms, having 3 digits there even makes some sense.